apps/web-giddh/src/app/theme/ng-virtual-select/sh-select.component.html

```diff
@@ -34,9 +34,10 @@
 
         <div (click)="showSearchSuggestion ? show($event) : '';" *ngIf="((!isOpen || !isFilterEnabled) && !multiple)">
             <ng-container *ngIf="selectedValues?.length">
-                <div *ngIf="!multiple">
+                <div *ngIf="!multiple" class="position-relative d-flex align-items-center selected-val-div">
                     <input autocomplete="disabled" type="text" readonly value="{{selectedValues[0].label}}"
-                        class="form-control selected-val cp" [attr.style]="borderConfiguration?.style" />
+                    class="form-control selected-val cp" [attr.style]="borderConfiguration?.style" />
+                    <i class="icon-arrow-down pd-r1"></i>
```

Comment:
Please check if it is required throughout the Giddh as this change will make icon appear on all the dropdown